.github/labeler.yml

```diff
@@ -16,7 +16,7 @@ integration:elasticsearch:
 
 integration:google-vertex:
```

Comment:
Should we change the labels too? 🤔

Reply:
we're going to have two naming conventions now:
- hyphens for package names: `google-vertex-haystack`
- underscores for folder names: `integrations/google_vertex/**`

I think it depends on what are you targeting with (for example in this case) a label: the folder in the source layout, or the "integration project"?

Reply:
Ah right, makes sense.